src/azure-cli/azure/cli/command_modules/keyvault/tests/hybrid_2018_03_01/test_keyvault_commands.py

```diff
@@ -311,6 +325,9 @@ def test_keyvault_secret(self, resource_group):
         self.kwargs['bak_file'] = bak_file
         self.cmd('keyvault secret backup --vault-name {kv} -n {sec} --file {bak_file}')
         self.cmd('keyvault secret delete --vault-name {kv} -n {sec}')
+        time.sleep(10)
```

Comment:
Not reliable.

Reply:
@qwordy Yes, just a value based on experience.